### ISSUE TITLE
Update unc0ver?

### ISSUE THREAD

**Issue by fikaa73** [LinusHenze/Fugu14#232]
Hey everyone
Stuff working good, but sometimes occasional reboot happens.
I am running unc0ver 7.0.1 on iPhone 12 14.5.1
Is there a way to update unc0ver untethered app?

**Comment by mohsinafrica** [comment]
Hey 
I am using iPhone 12pro 14.6
Is there any way to jailbreak for this

**Comment by fikaa73** [comment]
Yes friend you can jailbreak with unc0ver, you just can’t use untether
Sideload unc0ver with altstore

**Comment by mohsinafrica** [comment]
No it doesn't support 
Please 
Help me
Out 
[[![image](https://user-images.githubusercontent.com/101071334/158098285-3c62b774-3dd4-47dd-baa5-983cb2d3a502.png)]()]()
